tailwind.config.js

```diff
@@ -17,6 +17,7 @@ export default {
         yellow: '#FFB347', // used for buttons, text, and border in outline buttons
         darkGreen: '#007185', // used for buttons, text, and border in outline buttons
         lightBlue: '#F5F9FA', // used for backgrounds of book cards, footer, etc.
+        footerBg: '#F5F9FA', // alias for the lightBlue color, specifically for the footer
```

Comment:
There is already a color `#F5F9FA`. It's called `lightBlue` in the same config file. You don't need to create a new footer color.